server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfiles.java

```diff
@@ -101,20 +101,20 @@ public QProfile findProfileByProjectAndLanguage(long projectId, String language)
     return projectLookup.findProfileByProjectAndLanguage(projectId, language);
   }
 
-  public void addProject(int profileId, long projectId) {
-    projectOperations.addProject(profileId, projectId, UserSession.get());
+  public void addProject(String profileKey, String projectUuid) {
```

Comment:
Yep, those 2 classes must go. I'll tackle this in the "debt removal" sprint, once all WS are OK.